src/cfnlint/registry.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+import json
+import logging
+import os
+import platform
+import subprocess
+from cfnlint.template import Template
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Registry(object):
```

Comment:
Is this file just leftover from renaming this class? Can this file be removed if so?
https://github.com/aws-cloudformation/cfn-lint/pull/2056#discussion_r657306850

Reply:
I didn't catch it while resolving the merge conflicts. Thank you, will remove it!